packages/rpm/postinstall.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+/sbin/ldconfig
```

Comment:
According to [this](https://docs.fedoraproject.org/en-US/packaging-guidelines/Scriptlets/#_default_shell) you can just run `ldconfig` and don't need a shebang. So better just make this `ldconfig`.

Just to prevent any problems with paths, non-existent binaries etc.